### PR TITLE
Montreal python

### PR DESCRIPTION
**Pull request by Daniel-at-github**
Fixes issue #2

### CODE REVIEW

**Comment by redapple** [comment]
Maybe these talks could be grouped by year? what do you think @Daniel-at-github ?

**Comment by Daniel-at-github** [comment]
In conferences it makes a lot of sense, its one every year, and the organizers call it by the year.
In meetups... it's more diffuse, and nobody labels it by the year.

I would love to hear the opinions of the Montreal Python members. I see in the talks @prooffreader, @charettes, @mlhamel, @guillaumep, @EiNSTeiN-, @kamalmarhubi and @mathben
If any one of them want to comment on this or review this before (or after) it publishes on http://pyvideo.org it will be more than wellcome.

**Comment by guillaumep** [comment]
@Daniel-at-github I like the idea! The data looks good, at least for my talks. It looks like a fine job! (and a lot of trouble to get there)

**Comment by charettes** [comment]
@Daniel-at-github it makes a lot of sense to me. Thank you for dealing with that 🙇 !

**Comment by Daniel-at-github** [comment]
To me it's a way to give back to the community :)

Please, tell to Montreal Python about this and ask they if they want to add to pyvideo, as part of the release process, every future meetup video.
Any contribution is appreciated.

**Comment by redapple** [comment]
@guillaumep , @charettes , I'm not quite sure what you mean by "lik[ing] the idea" and "mak[ing] a lot of sense":
do you mean that you would prefer having meetup talks split by year (and having one "event" per year display on pyvideo.org, e.g. "Montreal Python 2015", "Montreal Python 2016" etc.),
or would you prefer to have all videos under the same "Montreal Python" event, and new talks being added to that umbrella event when they get published?

**Comment by kamalmarhubi** [comment]
How are you coming up with the licensing?

File: montreal-python/videos/lording-over-your-minions-with-salt-mp36.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      "label": "group web",
+      "url": "https://montrealpython.org"
+    },
+    {
+      "label": "MP36",
+      "url": "http://sflx.ca/mp36"
+    },
+    {
+      "label": "Kamal Marhubi",
+      "url": "https://github.com/kmrhb"
```
Comment (kamalmarhubi) [inline montreal-python/videos/lording-over-your-minions-with-salt-mp36.json]: my account was renamed to kamalmarhubi

**Comment by guillaumep** [comment]
@redapple I would think publishing all videos under the same umbrella event would make sense, as Montreal Python is a monthly event.

File: montreal-python/videos/lording-over-your-minions-with-salt-mp36.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "copyright_text": "Creative Commons Attribution license (reuse allowed)",
+  "description": "Montreal, April 8, 2013 -- Python developer Kamal Marhubi presents Salt -- a remote execution and configuration management tool written in Python. This talk is an introduction to salt, focusing on the configuration management aspects. it covers using salt with Vagrant as an alternative to Chef or Puppet. Kamal also demos using it to build up and control a machine on EC2 (network allowing).\n\nUseful links:\n\n* salt: http://sflx.ca/salt\n* Kamal: https://github.com/kmrhb\n* MP36: http://sflx.ca/mp36\n* Montreal Python: http://montrealpython.org\n* Savoir-faire Linux: http://www.savoirfairelinux.com",
```
Comment (kamalmarhubi) [inline montreal-python/videos/lording-over-your-minions-with-salt-mp36.json]: my account was renamed to kamalmarhubi

Comment (Daniel-at-github) [reply montreal-python/videos/lording-over-your-minions-with-salt-mp36.json]: Done in 9f35fce

File: montreal-python/videos/lording-over-your-minions-with-salt-mp36.json (hunk 1 above)
Comment (Daniel-at-github) [reply montreal-python/videos/lording-over-your-minions-with-salt-mp36.json]: Done in 9f35fce

File: montreal-python/videos/lording-over-your-minions-with-salt-mp36.json (hunk 2 above)
Comment (Daniel-at-github) [reply montreal-python/videos/lording-over-your-minions-with-salt-mp36.json]: If you edit it in the GitHub editor, I think, then it becomes a pull request to my pull request.

Lowering the work and entry barrier is a main goal in pyvideo :)

**Comment by redapple** [comment]
@kamalmarhubi , I believe the license comes directly from what YouTube says in the metadata.
Should another license be used instead?

**Comment by mlhamel** [comment]
I confirmed, @redapple, we don't really have the notion of season then i would keep everything as is.

Regarding the license, it's as @redapple mentioned in the metadata already, and I think it's always [Creative Common](https://support.google.com/youtube/answer/2797468)

File: montreal-python/category.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "title": "Montreal Python"
```
Comment (mlhamel) [inline montreal-python/category.json]: We habitually use `Montréal-Python` for every language

File: montreal-python/videos/00000.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  "related_urls": [
+    {
+      "label": "group web",
+      "url": "https://montrealpython.org"
+    }
+  ],
+  "speakers": [],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/tPDhu-PacfU/maxresdefault.jpg",
+  "title": "00000",
```
Comment (mlhamel) [inline montreal-python/videos/00000.json]: Since we've never add the proper title i'm guessing we will be able to update this in a subsequent PR ?

**Comment by mlhamel** [comment]
If no one is having any other comment, I'm gonna merge this. 

I would prefer tweaking the data into subsequent PR (for those talks without any title or speakers)

**Comment by redapple** [comment]
@mlhamel, done.

**Comment by Daniel-at-github** [comment]
Welcome to the colaborators @mlhamel 
Feel free to improve Montréal-Python data (or any other) adding an issue (or a pull request) and/or assigning it to yourself (to avoid unintentional duplicate reviewing).

And tell of pyvideo in Montréal-Python :innocent:

**Pull request merged**